.travis.yml

```diff
@@ -1,6 +1,7 @@
 ---
 # send to travis container infrastructure
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure
+dist: precise
 sudo: false
```

Comment:
is there a story for bumping to trusty/`sudo: true`? i think that's the only real change most people had to make?

Reply:
well the real problem is that psql 9.1 is `precise` only -- regardless of `sudo`

https://docs.travis-ci.com/user/database-setup/#PostgreSQL

so the story would be to bump to psql 9.2 -- i can make that story